guides/source/active_record_callbacks.md

```diff
@@ -435,9 +723,9 @@ class Author < ApplicationRecord
 end
 ```
 
-If a `before_add` callback throws `:abort`, the object does not get added to
-the collection. Similarly, if a `before_remove` callback throws `:abort`, the
-object does not get removed from the collection:
+If a `before_add` callback throws `:abort`, the object does not get added to the
+collection. Similarly, if a `before_remove` callback throws `:abort`, the object
+does not get removed from the collection:
```

Comment:
Should this sentence move below the code example since this position implies that the code will show this example, whereas it shows the code reflecting the first sentence.